documentation/docs/scenarios/abapEnvironmentAddons.md

```diff
@@ -70,9 +86,9 @@ The pipeline responsible for building ABAP add-ons has been created specifically
 
 The pipeline configuration is done in a git repository (for example on GitHub). This repository needs to be accessed by the Jenkins Server. If the repository is password protected, the user and password (or access token) should be stored in the Jenkins Credentials Store (Manage Jenkins -> Manage Credentials).
 
-#### Delivery Tools
+#### Add-on Assembly Kit as a Service (=AAKaaS)
```

Comment:
I would propose to discuss this with the AAKaaS colleagues, as they did not want to mention the term AAKaaS initially.

Reply:
Hi Daniel, yes makes sense. I think there were also the terms deploy tools and delivery tools which could also make sense. But then we should unify the wording in the scenario documentation and the piper steps where it's referred to. What I like about "AAKaaS" is the positioning as a "cloud service" instead of a system.

Reply:
Hi Daniel, I changed is so that we keep the delivery/deployment tools terminology: see https://github.com/SAP/jenkins-library/pull/2230/files#r514368916